satpy/readers/hrit_jma.py

```diff
@@ -317,6 +318,6 @@ def calibrate(self, data, calibration):
             res = xr.DataArray(res,
                                dims=data.dims, attrs=data.attrs,
                                coords=data.coords)
-        res = res.where(data > 0)
+        res = res.where(data < 65535)
```

Comment:
Shouldn't we mask negative values, too?

Reply:
that's the raw counts, which are uint16, so they don't have negative values.
Masking negative reflectances is another (hairy) topic :)

Reply:
Ok, agreed